File: Java/1672_Richest_Customer_Wealth.java
class Solution {
    public int maximumWealth(int[][] accounts) {
        int result = Integer.MIN_VALUE;
        for(int i = 0; i < accounts.length; i++){
            int sum = 0;
            for(int j = 0; j < accounts[i].length; j++){
                sum += accounts[i][j];
            }
            result = Math.max(result, sum);
        }
        return result;
    }
}

File: Python3/1672_Richest_Customer_Wealth.py
class Solution:
    def maximumWealth(self, accounts: List[List[int]]) -> int:
        return max([sum(account) for account in accounts])
        """
        t = 0;
        result = 0;
        for account in accounts:
            t = 0
            for w in account:
                t += w
            if t > result:
                result = t
        return result
        """
